browser-only.js

```diff
@@ -13,6 +13,9 @@ require('console-polyfill');
 
 require('whatwg-fetch');
 
+// KeyboardEvent.key shim
+require('shim-keyboard-event-key');
```

Comment:
this is failing travis because `KeyboardEvent` doesn't exist; I think you need to move it into the "it's a browser" block below.